src/cdk/drag-drop/drag-utils.ts

```diff
@@ -51,6 +57,35 @@ export function transferArrayItem<T = any>(currentArray: T[],
   }
 }
 
+/**
+ * Copies item from one array to another. Modifying transferArrayItem from cdk utils found here
+ *
+ * https://github.com/angular/material2/blob/master/src/cdk/drag-drop/drag-utils.ts
```

Comment:
Remove the "Modifying transferArrayItem from cdk utils found here https://github.com/angular/material2/blob/master/src/cdk/drag-drop/drag-utils.ts" since these are the docs that will be shown on the site.